pki/util.go

```diff
@@ -636,18 +645,17 @@ func getOUName(certName string) string {
 	}
 }
 
-func getCertFromFile(certDir string, fileName string) (*x509.Certificate, error) {
-	var certificate *x509.Certificate
+func getCertsFromFile(certDir string, fileName string) ([]*x509.Certificate, error) {
```

Comment:
I'm guessing we're ignoring the error here because it's expected, but I'm wondering if we should actually check it in case it's something unexpected. Either that or we can remove the `error` return value since it's always `nil`. Sorry, I know this isn't your code and you're not really changing anything here, but I figure we might want to take this opportunity to fix it.

Reply:
There is one case in the function where we return a real error (lines 653-655). I guess GH collapses those lines?

Reply:
I'm willing to support any changes that may be necessary, but from an initial glance this appears to be correct. As @jiaqiluo identified we do return an error when we can't parse the certificates from the file.
Are there any objections to me closing off this conversation?